code/modules/reagents/chemistry/recipes/drugs.dm

```diff
@@ -47,3 +47,10 @@
 	results = list(/datum/reagent/drug/happiness = 4)
 	required_reagents = list(/datum/reagent/nitrous_oxide = 2, /datum/reagent/medicine/epinephrine = 1, /datum/reagent/consumable/ethanol = 1)
 	required_catalysts = list(/datum/reagent/toxin/plasma = 5)
+
+/datum/chemical_reaction/ketamine
+	name = "Ketamine"
+	id = /datum/reagent/drug/ketamine
+	results = list(/datum/reagent/drug/ketamine = 3)
+	required_reagents = list(/datum/reagent/medicine/morphine = 3, /datum/reagent/toxin/chloralhydrate = 3, /datum/reagent/toxin/fentanyl = 3, /datum/reagent/medicine/epinephrine =3)
```

Comment:
Why have it be 3 parts of everything instead of 1 part

Reply:
To encourage batch production. Also meant to require a larger amount for production.

Reply:
I believe the recipe to result ratio is the exact same. If you wanted to increase the amount of required reagents to give a single unit of ketamine, just drop the result amount a bit :)

Reply:
Unless I misunderstood that means instead of a per unit(such as with oil), it takes 12 and collapses into 3

Reply:
I'm not _entirely_ sure what you mean but I'll explain it :D 

in this case you've got a 12:3 recipe:result ratio, I was just pointing out how this is the same as a 4:1 ratio.

3 morphine  + 3 chloral + 3 fenatyl + 3 epi = 3 ketamine
==
1 morphine + 1 chloral + 1 fenatyl + 1 epi = 1 ketamine
tldr; there's no real reason to have it 3 for everything _unless_ you wanted to make the initial reagent amount require 12 instead of 4. If that's the case then your current code works fine, although I personally don't see why you'd want it to do that xD